client/extensions/wp-super-cache/components/easy.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import SectionHeader from 'components/section-header';
+
+class Easy extends Component {
+	state = {
+		checkedRadio: 'off',
+	};
+
+	handleRadioChange = event => this.setState( { checkedRadio: event.currentTarget.value } );
+
+	render() {
+		const { translate } = this.props;
+		const { checkedRadio } = this.state;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Caching' ) }>
+					<Button
+						compact={ true }
+						primary={ true }
+						type="submit">
+							{ translate( 'Update Status' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Status Update' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpsc-easy-settings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).